src/forklift/lift.py

```diff
@@ -77,17 +78,41 @@ def process_pallets(pallets):
                 log.debug('shipped pallet %s', seat.format_time(clock() - start_seconds))
             except Exception as e:
                 pallet.success = (False, e.message)
+
                 log.error('error shipping pallet: %s for pallet: %r', e.message, pallet, exc_info=True)
 
-        reports.append(pallet.get_report())
 
-    return reports
+def copy_data(pallets, copy_destinations):
+    '''pallets: Pallets[]
+
+    Loop over all of the pallets and extract the distinct copy_data workspaces.
+    Then loop over all of the copy_data workspaces and copy them to copy_destinations as defined in the config.'''
+    copy_workspaces = []
+    for pallet in pallets:
+        if pallet.is_ready_to_ship():
+            copy_workspaces = copy_workspaces + pallet.copy_data
+    copy_workspaces = set(copy_workspaces)
+
+    for source in copy_workspaces:
+        for destination in copy_destinations:
+            destination_workspace = path.join(destination, path.basename(source))
+
+            log.info('copying {} to {}...'.format(source, destination_workspace))
+            start_seconds = clock()
+            try:
+                if path.exists(destination_workspace):
+                    shutil.rmtree(destination_workspace)
+                shutil.copytree(source, destination_workspace)
+                log.info('copy successful in %s', seat.format_time(clock() - start_seconds))
+            except Exception as e:
+                pallet.success = (False, str(e))
+                log.error('there was an error copying {} to {}: {}'.format(source, destination_workspace, e))
```

Comment:
with logging it is recommended to use the printf format instead of format. It's also a good idea to add the [exec info](https://github.com/agrc/forklift/blob/ebf9cdec2ad4d5add933dadd9c8c3b545a74bfee/src/forklift/lift.py#L67).